spec/posthog/feature_flag_spec.rb

```diff
@@ -1336,19 +1336,169 @@ class PostHog
       end
     end
 
-    it 'nil property value with all operators' do
+    # def test_none_property_value_with_all_operators(self):
```

Comment:
some leftover copy and paste (ignore if you're still WIPing it)

Reply:
nah this one's ready, will remove thanks!